pkg/reconciler/certificate/resources/cert_manager_certificate_test.go

```diff
@@ -83,6 +84,9 @@ func TestMakeCertManagerCertificate(t *testing.T) {
 				Kind: "ClusterIssuer",
 				Name: "Letsencrypt-issuer",
 			},
+			SecretTemplate: &cmv1.CertificateSecretTemplate{
```

Comment:
No harm to add this by default. I think its too much to have a flag for filtering in this component.